app/templates/create_buyer/create_buyer_account.html

```diff
@@ -20,9 +20,13 @@
 
 {% block mainContent %}
 
-{% with lede = "There was a problem with the details you gave for:" %}
-  {% include "toolkit/forms/validation.html" %}
-{% endwith %}
+<h1 class="govuk-heading-l">
+  Create a buyer account
+</h1>
+
+<p class="govuk-body-l">
+  You must be employed by, or represent, a public sector organisation.
+</p>
```

Comment:
This content was moved in a previous PR but appears to have come back in a merge conflict

```suggestion
```